src/JSInterop/Microsoft.JSInterop/src/JSRuntime.cs

```diff
@@ -7,9 +7,9 @@
 namespace Microsoft.JSInterop
 {
     /// <summary>
-    /// Provides mechanisms for accessing the current <see cref="IJSRuntime"/>.
+    /// Abstract base class for a JavaScript runtime.
     /// </summary>
-    public static class JSRuntime
+    public abstract partial class JSRuntime : IJSRuntime
```

Comment:
Git's diff gets messy if I rename `JSRuntimeBase.cs` -> `JSRuntime.cs`. I'll clean it up before this is merged